usagov/vote-gov
# Vote 226 - Cypress Install

- This PR will add Cypress to vote.gov 
- Pr sets up cypress and testing scripts for cypress to run 

to test 
- pull branch to local 
- run `npm install`
- run `npm run start`
- spilt terminal
- run `npm run cy:open`
this will open the cypress runner and click on test to ensure the cypress tests are running
